package.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/ossf/package-feeds/blob/main/package.schema.json",
+    "title": "Package Schema Version 1.0",
+    "description": "The package representation as outputted by a ScheduledFeed",
+    "examples": [
```

Comment:
This seems strange as a list of values for the potential fields? Is this supposed to be a list of objects with key/values?

Reply:
I went off https://json-schema.org/understanding-json-schema/reference/generic.html#annotations for this, which doesn't specify it it should be a list of tuples. I've not got another reference point/real example to hand though

Reply:
Ah it seems like this `examples` field belongs in each of the field objects? This would make more sense for a list of examples. Works perfectly for our `type` value too!

Reply:
Pushed an update to have an examples annotation on a per property basis, and also an extra test for tighter control on additional properties. How does it look now @Qinusty ?